src/main.c

```diff
@@ -2740,7 +2800,7 @@ se_debug_tool_desc_t gb_debug_tools[]={
   {ICON_FK_VOLUME_UP, ICON_FK_VOLUME_UP " PSG",se_psg_debugger},
   {ICON_FK_DELICIOUS, ICON_FK_DELICIOUS " Tile Map",gb_tile_map_debugger},
   {ICON_FK_TH, ICON_FK_TH " Tile Data",gb_tile_data_debugger},
-  {ICON_FK_AREA_CHART, ICON_FK_AREA_CHART " Emulator Stats",se_draw_emu_stats, .allow_hardcore=true},
+  {ICON_FK_AREA_CHART, ICON_FK_AREA_CHART " Emulator Stats",se_draw_emu_stats, .allow_hardcore=true}, // TODO: why does emerald unlock an achievement on boot
```

Comment:
What is meant by this?

Reply:
The emerald game also had a similar bug with pokemon black again because of how retroachievements reads from memory, this comment should also be deleted it works correctly now